CODE_OF_CONDUCT.md

```diff
@@ -62,6 +62,7 @@ Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the maintainers responsible for enforcement at:
 
 alexvonme at matrix dot org
+(This is not an email address, you may reach it after opening an account in matrix)
```

Comment:
please add a link to it then.

People are lazy :grin:

Reply:
@alexvonme what about this?

Reply:
I'll explain this one in the other platform we chat in.

Reply:
> please add a link to it then.

I don't think we need to. People link to, for example, their mastodon accounts without giving links to mastodon all the time. It's expected that people will end up in the designated network anyway when seeing a link like ``@example@mastodon.social``.

It's redundant to add links when the account starts with ``@`` since people assume the later part is a link at this point.